packages/react/cypress/component/auto/form/AutoFormInputLabels.cy.tsx

```diff
@@ -1,15 +1,26 @@
 /* eslint-disable jest/valid-expect */
 import React from "react";
 import { PolarisAutoInput } from "../../../../src/auto/polaris/inputs/PolarisAutoInput.js";
+import { makeShadcnAutoInput } from "../../../../src/auto/shadcn/inputs/ShadcnAutoInput.js";
 import { humanizeCamelCase } from "../../../../src/utils.js";
 import { api } from "../../../support/api.js";
 import { describeForEachAutoAdapter } from "../../../support/auto.js";
 
+import { Button } from "../../../../spec/auto/shadcn-defaults/components/Button.js";
+import { Checkbox } from "../../../../spec/auto/shadcn-defaults/components/Checkbox.js";
+import { Input } from "../../../../spec/auto/shadcn-defaults/components/Input.js";
+import { Label } from "../../../../spec/auto/shadcn-defaults/components/Label.js";
+
 const AutoInput = (props: { suiteName: string; field: string; label?: string }) => {
   if (props.suiteName === "Polaris") {
     return <PolarisAutoInput {...props} />;
   }
 
+  if (props.suiteName === "Shadcn") {
+    const ShadcnAutoInput = makeShadcnAutoInput({ Input: Input, Label: Label, Checkbox: Checkbox, Button: Button });
```

Comment:
nit: can you move this outside of this function? Otherwise, it's going to re-create the input component every render and for every test, which is kinda slow, and also not how production uses of this stuff will work so I think it will test it a bit differently. the factory functions should generally be invoked in the module scope